engines/cge2/vmenu.cpp

```diff
@@ -25,16 +25,22 @@
  * Copyright (c) 1994-1997 Janusz B. Wisniewski and L.K. Avalon
  */
 
+#include "common/config-manager.h"
+#include "common/text-to-speech.h"
 #include "cge2/text.h"
 #include "cge2/vmenu.h"
 #include "cge2/events.h"
 
 namespace CGE2 {
 
+const char *_iWantToStopPlayingAlready = "";
+const char *_thisIsAMistakeKeepPlaying = "";
```

Comment:
Those are not class member variables, and thus should not start with an underscore.

Reply:
Also as I noted below I don't think you actually need those variables.